runtime/basilisk/src/migrations.rs

```diff
@@ -0,0 +1,57 @@
+use super::*;
+
+/// Migrate the Uniques pallet storage to v1
+pub struct MigrateUniquesPallet;
+impl OnRuntimeUpgrade for MigrateUniquesPallet {
+	fn on_runtime_upgrade() -> Weight {
+		pallet_uniques::migration::migrate_to_v1::<Runtime, _, Uniques>()
+	}
+}
+
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+pub struct OnRuntimeUpgradeMigration;
+impl OnRuntimeUpgrade for OnRuntimeUpgradeMigration {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		frame_support::log::info!("PreMigrate Marketplace Pallet start");
+		pallet_marketplace::migration::v1::move_old_storage::pre_migrate::<Runtime>();
+		frame_support::log::info!("PreMigrate Marketplace Pallet end");
+
+		frame_support::log::info!("PreMigrate NFT Pallet start");
+		pallet_nft::migration::v1::pre_migrate::<Runtime>();
+		frame_support::log::info!("PreMigrate NFT Pallet end");
+
+		Ok(())
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		let mut weight: Weight = 0;
+
+		frame_support::log::info!("Migrate Uniques Pallet start");
+		weight = weight.saturating_add(<MigrateUniquesPallet as OnRuntimeUpgrade>::on_runtime_upgrade());
+		frame_support::log::info!("Migrate Uniques Pallet end");
+
+		frame_support::log::info!("Migrate Marketplace Pallet start");
+		weight = weight.saturating_add(pallet_marketplace::migration::v1::move_old_storage::migrate::<Runtime>());
+		frame_support::log::info!("Migrate Marketplace Pallet end");
+
+		frame_support::log::info!("Migrate NFT Pallet start");
+		weight = weight.saturating_add(pallet_nft::migration::v1::migrate::<Runtime>());
+		frame_support::log::info!("Migrate NFT Pallet end");
+
+		weight
+	}
```

Comment:
Have we tested that this fits in a block with the production data?

Reply:
Could you please have a look at this [commit](https://github.com/galacticcouncil/Basilisk-node/pull/564/commits/574268219097ba23748a6ba38e81001e02490896)?

Reply:
So we're planning to run a test after merging? Or you have run with try-runtime and production data already and it's fine?

Reply:
I tested it with try-runtime feature against the production data and it consumed less than 2% of the total block weight.